tutorials/relay.md

```diff
@@ -201,20 +177,18 @@ docker run --rm -ti -v $PWD:$PWD \
     --relayUrl https://<your hostname for the relay>/gsn1 \
     -m `pwd`/pass12
 ```
-+
 {% hint style="note" %}
-### NOTE:
+##### NOTE:
```

Comment:
I deliberately put it smaller, otherwise it looks HUGE on gitbook site (try to open it in its temporary site: docs2.opengsn.org